apps/cloud_composer/include/pcl/apps/cloud_composer/items/fpfh_item.h

```diff
@@ -43,6 +43,14 @@
 #include <pcl/apps/cloud_composer/items/cloud_composer_item.h>
 #include <pcl/visualization/pcl_plotter.h>
 
+#include <vtkVersion.h>
+
+#if VTK_MAJOR_VERSION > 8
+#include <QVTKOpenGLNativeWidget.h>
```

Comment:
Use the vtk compatibility header and type that you've added and used in other places